doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,881 @@
+
+    * Removed:
+
+      * Overview diagram in :ref:`nrfxlib:nrf_modem`.
+      * AT socket page.
+      * Extensions page.
+
+  * :ref:`nrf_security`:
+
+    * Added experimental support for PSA crypto drivers in :ref:`nrf_security`.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+Unity
+-----
+
+* Fixed a bug that resulted in the mocks for some functions not having the ``__wrap_`` prefix. This happened for functions declared with whitespaces between identifier and parameter list.
+
+HID Configurator for nRF Desktop
+--------------------------------
+
+* Updated:
+
+  * The script to recognize the bootloader variant as a DFU module variant for the configuration channel communication. The new implementation is backward compatible - the new version of the script checks for module name and acts accordingly. See the Performing DFU section in :ref:`nrf_desktop_config_channel_script` documentation.
+
+  * Documentation on setting up the dependencies for Debian/Ubuntu/Linux Mint operating system.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``1eedec3e79``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added:
+
+  * Check of reset address in the incoming image validation phase. See :kconfig:`CONFIG_MCUBOOT_VERIFY_IMG_ADDRESS`.
+
+* Updated:
+
+  * Always call :c:func:`sys_clock_disable` in main since the empty :c:func:`sys_clock_disable` callback is provided if the platform does not support system clock disable capability.
+  * Bootutil library to close flash_area after failure to read swap status information.
+  * Allow image header bigger than 1 KB for encrypted images.
+  * Zephyr:RSA to use a smaller SHA256 implementation.
+
+* Fixed:
+
+  * Zephyr/boot_serial_extension build failure on :c:macro:`MCUBOOT_LOG_MODULE` when LOG is disabled.
+  * Status offset calculation in case of scratch area.
+  * Build with image encryption using RSA (issue introduced by migration to MbedTLS 3.0.0).
+  * Support for single application with serial recovery.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``3f82656``, plus some |NCS| specific additions.
+
+For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline 3f82656 ^v2.6.0-rc1-ncs1
```

Comment:
Due to the high number of fromtree commits, I don't think this list is particularly useful.

It can't be used to answer any interesting questions that a customer might have that I can think of.

E.g. to answer "Is zephyr commit X included?" or "Which commits are included?"
you really need to look at the fromlist commits as well.

I suggest we replace it with a command that also prints out our own patches.

Reply:
oh, I guess in tandem with the below command it is OK